docs/user_guide/data_types_and_io/dataclass.md

```diff
@@ -12,11 +12,11 @@ Flytekit uses the [Mashumaro library](https://github.com/Fatal1ty/mashumaro)
 to serialize and deserialize dataclasses.
 
 :::{important}
-If you're using Flytekit version below v1.10, you'll need to decorate with `@dataclass_json` using
-`from dataclasses_json import dataclass_json` instead of inheriting from Mashumaro's `DataClassJSONMixin`.
-
-If you're using Flytekit version >= v1.11.1, you don't need to decorate with `@dataclass_json` or
+If you're using Flytekit version below v1.11.1, your dataclass will need to decorate with `@dataclass_json` or
```

Comment:
Just to clarify, this means that if you are using Flytekit < 1.11.1, you either need to decorate your dataclass with `@dataclass_json` or create a class that inherits `DataClassJSONMixin`, but that you don't need to do both? If you inherit from `DataClassJSONMixin`, do you need to do `from mashumaro.mixins.json import DataClassJSONMixin` as well?

Reply:
yes!